.github/workflows/ci-example-ios.yml

```diff
@@ -48,6 +48,9 @@ jobs:
           bundler-cache: true
           ruby-version: '2.7'
 
+      - name: Install SwiftLint
```

Comment:
Not sure if this was already mentioned in the docs as something the iOS side needs for this library, but if it isn't, you might want to add it to the contribution guide or maybe break it out of this PR

Reply:
I suspect swiftlint was preinstalled on older GitHub macos images. When I bumped the image to `macos-14` this was necessary (but it was used before).

Reply:
Sounds good! Resolved!